### PR TITLE
Feature/12875161 add save only at pictures folder mode

### PR DESCRIPTION
**Pull request by ericcleao**

### CODE REVIEW

File: cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt [hunk 1]
```diff
@@ -0,0 +1,513 @@
...
+                    navigationCamera.paddingRight,
+                    navigationCamera.bottom + convertDpToPixels(48))
+        }
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        mSaveOnlySnackbar = Snackbar.make(mCameraFragmentBinding.root,
+                getString(R.string.camera_snackbar_save_only_mode_message),
```
Comment (denisvieiradev) [inline cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt]: change this strings to be dinamically, texts should be provided by those who use the library

File: cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+                Snackbar.LENGTH_SHORT)
+        mStantSaveModeSnackbar = Snackbar.make(mCameraFragmentBinding.root,
+                getString(R.string.camera_snackbar_stant_save_mode_message),
```
Comment (denisvieiradev) [inline cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt]: change this strings to be dinamically, texts should be provided by those who use the library

File: cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt [hunk 3]
```diff
@@ -0,0 +1,513 @@
...
+        updateCounters()
+    }
+
+    private fun setButtonsClick(closeButton: ImageView, changeSavePicturesReason: ImageView,
+                                captureButton: ImageButton, openGalleryButton: ImageView, savePhotosButton: ImageView) {
+        closeButton.setOnClickListener { closeCamera() }
+
+        changeSavePicturesReason.setOnClickListener{
+            mSaveMode = if (mSaveMode.mode == STANT_MODE) {
+                changeSavePicturesReason.setImageDrawable(ContextCompat.getDrawable(context!!, R.drawable.ic_stant_disabled))
```
Comment (denisvieiradev) [inline cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt]: this icons can be provided by those use the library, add dinamical variable to icon

File: cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt (hunk 2 above)
Comment (ericcleao) [reply cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt]: Fixed

File: cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt (hunk 3 above)
Comment (ericcleao) [reply cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt]: Fixed

File: cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt (hunk 1 above)
Comment (ericcleao) [reply cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraFragment.kt]: Fixed

**Review by denisvieiradev — APPROVED** [review]

**Pull request merged**